### ISSUE TITLE
Don't indent multiple string literals in msgid/msgid_plural/msgstr

### ISSUE THREAD

**Issue by whatyouhide** [elixir-lang/gettext#132]
I think we should stop indenting strings when `msgid`/`msgid_plural`/`msgstr` have multiple strings. That is, right now when dumping we indent as:

```po
msgid "foo"
  "bar"
```

but I think we should start doing

```po
msgid "foo"
"bar"
```

This is to accommodate what appears to be a convention between PO editors (such as http://transifex.com). It is also show in the [gettext docs](https://www.gnu.org/software/gettext/manual/gettext.html#PO-Files):

> Each of untranslated-string and translated-string respects the C syntax for a character string, including the surrounding quotes and embedded backslashed escape sequences. When the time comes to write multi-line strings, one should not use escaped newlines. Instead, a closing quote should follow the last character on the line to be continued, and an opening quote should resume the string at the beginning of the following PO file line. For example:

```po
msgid ""
"Here is an example of how one might continue a very long string\n"
"for the common case the string represents multi-line output.\n"
```

> In this example, the empty string is used on the first line, to allow better alignment of the H from the word ‘Here’ over the f from the word ‘for’. In this example, the msgid keyword is followed by three strings, which are meant to be concatenated. Concatenating the empty string does not change the resulting overall string, but it is a way for us to comply with the necessity of msgid to be followed by a string on the same line, while keeping the multi-line presentation left-justified, as we find this to be a cleaner disposition.

As you can see from this issue, the GitHub syntax highlighting is not even highlighting our indentation as correct and paints the string red.

**Comment by josevalim** [comment]
:+1:

**Comment by whatyouhide** [comment]
Closed in 9b47ff0a04b1d551aadcd4f6d561b36d5c3b47f4.
